conf/sample.ini

```diff
@@ -799,6 +799,18 @@
 # Should be set for user-assigned identity and should be empty for system-assigned identity
 ;managed_identity_client_id =
 
+# Azure AD Workload Identity
```

Comment:
```suggestion
# Azure AD Workload Identity
# Specifies whether workload identity authentication should be enabled in datasources
# that support it. For more details on Workload Identity, review documentation here:
# https://azure.github.io/azure-workload-identity/docs/
```

Perhaps add this comment in all the `ini` files

Reply:
Updated, please check wordings